### ISSUE TITLE
Projects with duplicate Titles causes a "DuplicatePermalinkOutputError"

### ISSUE THREAD

**Issue by jhung** [inclusive-design/idrc#487]
## Describe the bug

If two project sub-pages have the same title like "About this project", upon saving the sub-page, a "DuplicatePermalinkOutputError" appears.

![image](https://user-images.githubusercontent.com/564762/222001046-eb45a386-bd3a-4672-ae26-6e6577b10ebe.png)

## To Reproduce

1. Create a project sub-page.
2. Give it a Title "Test" and publish it.
3. Create another project sub-page (can be a sub-page of the same project or a different one).
4. Give it a Title "Test" and publish it.
5. See the error occur

## Expected behavior

There can be a couple different ways to solve this:
* Preferred method: Generate permalinks for sub-pages prefixed with the project name (maybe the short name)
* Less-preferred method: Append a unique ID or number to the end of duplicate permalinks.
* Least-preferred method: Throw an error on submission asking the user to use a different Title as it's already used.

**Comment by jhung** [comment]
When trying to run `npm run build` you will receive this error if two projects share the same Title.

```
[build:eleventy] [11ty] Problem writing Eleventy templates: (more in DEBUG output)
[build:eleventy] [11ty] Output conflict: multiple input files are writing to `dist/projects/test-sub-sub-page-2/index.html`. Use distinct `permalink` values to resolve this conflict.
[build:eleventy] [11ty]   1. ./src/projects/test-sub-sub-page-2-1.md
[build:eleventy] [11ty]   2. ./src/projects/test-sub-sub-page-2.md (via DuplicatePermalinkOutputError)
[build:eleventy] [11ty]
[build:eleventy] [11ty] Original error stack trace: (Repeated output has been truncated…)
[build:eleventy] [11ty]     at TemplateMap.checkForDuplicatePermalinks (C:\Users\jhung\development\idrc\node_modules\@11ty\eleventy\src\TemplateMap.js:656:13)
[build:eleventy] [11ty]     at TemplateMap.cache (C:\Users\jhung\development\idrc\node_modules\@11ty\eleventy\src\TemplateMap.js:365:10)
[build:eleventy] [11ty]     at processTicksAndRejections (internal/process/task_queues.js:95:5)
[build:eleventy] [11ty]     at async TemplateWriter._createTemplateMap (C:\Users\jhung\development\idrc\node_modules\@11ty\eleventy\src\TemplateWriter.js:244:5)
[build:eleventy] [11ty]     at async TemplateWriter.generateTemplates (C:\Users\jhung\development\idrc\node_modules\@11ty\eleventy\src\TemplateWriter.js:277:5)
[build:eleventy] [11ty]     at async TemplateWriter.write (C:\Users\jhung\development\idrc\node_modules\@11ty\eleventy\src\TemplateWriter.js:323:23)
[build:eleventy] [11ty]     at async Eleventy.executeBuild (C:\Users\jhung\development\idrc\node_modules\@11ty\eleventy\src\Eleventy.js:1003:13)
[build:eleventy] [11ty] Copied 2 files / Wrote 0 files in 0.87 seconds (v1.0.2)
```

**Comment by greatislander** [comment]
I think this could also be resolved by #631 (or perhaps that's what you mean in your preferred method, @jhung).

**Comment by greatislander** [comment]
I tested and confirmed that #511 resolved this issue.
